libnetwork/resolvconf/resolvconf.go

```diff
@@ -1,22 +1,16 @@
 // Package resolvconf provides utility code to query and update DNS configuration in /etc/resolv.conf
+//
+// Its functions are implemented in terms of 'libnetwork/internal/resolvconf', which is specialised
+// for use by libnetwork's use-cases.
```

Comment:
```suggestion
```

What are you doing? The package doc comment is externally visible! Documenting implementation details makes them part of the API contract!

Reply:
Oh, right. Removed.